operators/multiclusterobservability/manifests/base/grafana/dash-k8s-apiserver.yaml

```diff
@@ -421,6 +427,7 @@ data:
           },
           "yaxes": [
             {
+              "$$hashKey": "object:258",
```

Comment:
what's the purpose to add $$hashKey?

Reply:
I believe the `$$hashkey` is added by Angular to keep track of your changes, that way it can update the DOM properly. I'm not sure why it was added here.

Reply:
Removed the `$$hashkey` value.